src/app/shared/services/platform.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Inject, Injectable, PLATFORM_ID } from '@angular/core'
+import { isPlatformBrowser, isPlatformServer } from '@angular/common'
+
+export interface IPlatformService {
+  readonly isBrowser: boolean
+  readonly isServer: boolean
+}
+
+@Injectable()
+export class PlatformService implements IPlatformService {
```

Comment:
Are you sure we need this service? I'd prefer not to use isBrowser/isServer checks whenever possible. 

There are separate main modules for browsers and server, in most cases it should be possible to use a different implementation of a service for server or browser, instead of explicitly checking whether you're running on the server or on the browser, which kind of defeats the purpose of Angular Universal 

For an example: see the TransferStateService for [browser](https://github.com/DSpace/dspace-angular/blob/4fbceab1b941f491c1d88e148a72f5608def00dc/src/modules/transfer-state/dspace-browser-transfer-state.service.ts) and [server](https://github.com/DSpace/dspace-angular/blob/4fbceab1b941f491c1d88e148a72f5608def00dc/src/modules/transfer-state/dspace-server-transfer-state.service.ts).

Reply:
It doesn't look like this is used anymore (which is good, thank you 🙂). Can you remove the service? Otherwise others may be tempted to use it.